docs/src/manual/input.rst

```diff
@@ -303,6 +303,8 @@ contains the following fields and values:
                     "spaceFilling": false,
                     // show atoms labels
                     "atomLabels": false,
+                    // use an environment property value for the atom labels
+                    "labelsProperty": false,
```

Comment:
I would rather turn `atomLabels` to a string, which can be `none`, `elements`, or one of the dataset property. We already do this for the atom color.